### PR TITLE
feat(cli): adding alert list and open commands

### PR DESCRIPTION
**Pull request by hazedav**
## Summary
Adding `alert list` and `alert open` commands

`alert list` uses time range flags (`--range`, `--start`, and `--end` consistent with lql)

## How did you test this change?
Integration testing

## Issue
https://lacework.atlassian.net/browse/ALLY-1183

### CODE REVIEW

**Comment by hazedav** [comment]
When reviewing this please see changes from 6 commits (as `lacework alert show` was encapsulated in a separate PR before being merged in here).  This can be achieved with a "shift select" in the "Files changed" menu, or this link should also work:
https://github.com/lacework/go-sdk/pull/961/files/b0aefd357fc34c7008bed52a8f051e9f4c77ed8f

File: api/alerts.go
Comment (afiune) [inline api/alerts.go:75]: Can we sort by severity instead?

File: cli/cmd/alert_list.go
Comment (afiune) [inline cli/cmd/alert_list.go:48]:
```suggestion
Or by specifying start and end flags.
```

File: cli/cmd/alert_list.go
Comment (afiune) [inline cli/cmd/alert_list.go:177]: Can we copy the table options from "events2?
```go
// easily add or remove borders to all event details tables
func eventsTableOptions() tableOption {
	return tableFunc(func(t *tablewriter.Table) {
		t.SetBorders(tablewriter.Border{
			Left:   false,
			Right:  false,
			Top:    true,
			Bottom: false,
		})
		t.SetColumnSeparator(" ")
	})
}
```

What do you think?

File: cli/cmd/alert_comment.go
Comment (afiune) [inline cli/cmd/alert_comment.go:33]: should we validate that the alert ID exists before trying to add a comment? that is, before typing the comment via the EDITOR.

**Review by afiune — APPROVED** [review]
tenor-207694606

File: api/alerts.go
Comment (hazedav) [reply api/alerts.go:75]: Sure thing.

File: cli/cmd/alert_list.go
Comment (hazedav) [reply cli/cmd/alert_list.go:48]: Good eye.

File: cli/cmd/alert_comment.go
Comment (hazedav) [reply cli/cmd/alert_comment.go:33]: This requires another round trip to the API.  We've gone back and forth with what sort of pre-validation we want to do on the client vs. server.  I will defer to your judgement here.

File: cli/cmd/alert_list.go
Comment (hazedav) [reply cli/cmd/alert_list.go:177]: Do you want all the alert tables to use these options?

Comment (afiune) [reply cli/cmd/alert_list.go:177]: It really depends, I wish we could have someone from UX to look at all these tables. When I did
`lacework event` I tried to use the terminal space as much as possible. I will leave it to your criteria
but use as much space as possible when it makes sense and when it doesn't make the output look
confusing, that is, when there are big long text.

File: cli/cmd/alert_comment.go
Comment (afiune) [reply cli/cmd/alert_comment.go:33]: I think that is what we did with `lacework query update <id>` 🤔 - I would suggest us doing that
extra API call to make sure the customer doesn't start writing the comment and then, find out
that the id was wrong.

Comment (hazedav) [reply cli/cmd/alert_comment.go:33]: I think this could also apply to `lacework alert close` so I will file a Jira for us to follow up on this as an enhancement.

Comment (hazedav) [reply cli/cmd/alert_comment.go:33]: https://lacework.atlassian.net/browse/ALLY-1261

File: api/alerts.go
Comment (hazedav) [reply api/alerts.go]: Fixed.

File: cli/cmd/alert_list.go
Comment (hazedav) [reply cli/cmd/alert_list.go]: Fixed.

File: cli/cmd/alert_list.go
Comment (hazedav) [reply cli/cmd/alert_list.go:173]: I'm going to create something for follow up on this.

**Pull request merged**
